docs/building.md

```diff
@@ -14,10 +14,16 @@ a number of hours to complete.
 ## Building the Rust Components
 
 *Complete this section before moving to the android/iOS build instructions.*
-
-1. Install Rust: install [via rustup](https://www.rust-lang.org/tools/install)
-1. Install your system dependencies: - install via the instructions below for [Linux](building.md#linux), [MacOS](building.md#macos) or [Windows](building.md#windows)
-1. Check dependencies, environment variables and test
+1. Make sure you cloned the repository:
+  ```shell
+    $ git clone https://github.com/mozilla/application-services (or use the ssh link)
```

Comment:
Please put the "or use the ssh link" part behind a `#` so that it is a valid comment, since this snippet is otherwise well-formed shell script.